nrf_security/Kconfig

```diff
@@ -0,0 +1,468 @@
+menu "Nordic Security"
+
+config NORDIC_SECURITY_BACKEND
+	bool
+	prompt "Use Nordic provided security backend"
+	depends on SOC_FAMILY_NRF
+	select NRFXLIB_CRYPTO
+	help
+	  Help message
+
+if NORDIC_SECURITY_BACKEND
+
+comment "Backend Selection"
+
+config CC310_BACKEND
+	bool "Use cc310 backend"
+	default y
+	depends on SOC_NRF52840
+	select NRF_CRYPTO_MULTIPLE_CHOICE_GROUP_0 if VANILLA_BACKEND
+	help
+	  Enable cc310 backend
+
+config VANILLA_BACKEND
+	bool "Use mbedtls backend"
+	default y if !(SOC_NRF52840 || SOC_NRF9160)
+	help
+	  Enable mbedtls vanilla backend
+
+# Group 0 is used to indicate when AES and ECC should use choice menu
+config NRF_CRYPTO_MULTIPLE_CHOICE_GROUP_0
+	bool
+
+# List of additional symbols to indicate cipher key sizes.
+# Currently selected based backend selection.
+config MBEDTLS_CIPHER_AES_256_ECB_C
+	bool
+
+config MBEDTLS_CIPHER_AES_256_CBC_C
+	bool
+
+config MBEDTLS_CIPHER_AES_256_CTR_C
+	bool
+
+config MBEDTLS_CIPHER_AES_256_CCM_C
+	bool
+
+menuconfig CUSTOM_MBEDTLS_CFG
+	bool
+	prompt "Use user provided mbedtls <config.h>"
+	depends on VANILLA_BACKEND
+
+if CUSTOM_MBEDTLS_CFG
+
+config CUSTOM_MBEDTLS_CFG_FILE
+	string "Custom mbed TLS configuration file"
+	default "custom-mbedtls-config.h"
+	help
+	  Use a custom mbed TLS configuration file. The nRF Connect SDK
+	  Security will generate a mbed TLS configuration file based on the
+	  selection of configuration options in Kconfig.
+	  However, if the generated configuration file needs additional
+	  adjustments, this setting can be used to provide a custom mbed TLS
+	  configuration file instead.
+
+endif # CUSTOM_MBEDTLS_CFG
+
+comment "Configuration section"
+
+menuconfig AES_CONFIGURATION
+	bool
+	prompt "AES Cipher Mode configuration"
```

Comment:
The prompt should tell what is the difference between enabling and disabling this config.

Reply:
Will fix